lib/action_account.rb

```diff
@@ -211,18 +211,23 @@ def run(args:, config:)
         end
 
         Analytics.new(
-          user: user, request: nil, session: {}, sp: nil,
+          user: user,
+          request: nil,
+          session: {},
+          sp: profile&.initiating_service_provider_issuer,
         ).fraud_review_rejected(
           success:,
           errors: analytics_error_hash,
           exception: nil,
-          profile_fraud_review_pending_at: profile_fraud_review_pending_at,
-          profile_age_in_seconds: profile_age_in_seconds,
+          profile_fraud_review_pending_at:,
+          profile_age_in_seconds: profile&.profile_age_in_seconds,
```

Comment:
(suggestion, non-blocking): I think it is worthwhile keeping the profile attr wrapping methods to streamline reading as `profile` attributes are inconsistently accessed with safe navigation (see line [281](https://github.com/18F/identity-idp/pull/11390/files?diff=split&w=1#diff-74ebdc3a9e3c25ee1f969fb4f34e75fedba69d99b6906ef805d9dbf9536bae8dR281) where we don't use safe navigation).